reference/api/rest.rst

```diff
@@ -0,0 +1,358 @@
+REST API
+========
+
+HTTP Verbs
+----------
+
+====== ===========
+Verb   Description
+====== ===========
+GET	   Used for retrieving a resource or a collection of resources.
+POST   Used for creating a resource.
+PATCH  Used for partially updating a resource.
+PUT	   Used for replacing a resource.
+DELETE Used for deleting a resource.
+====== ===========
+
+Pagination
+----------
+
+All responses containing a collection of resources will be paginated by 10 items by default.
+
+.. warning::
+    You cannot request more than 100 resources at the same time.
+    An error with code 400 will be thrown if the limit is bigger than 100.
+
+TODO: Ajouter un lien vers un cookbook pour modifier la limite.
+
+The response will respect this structure, even if there is no item to return.
+
+.. code-block:: json
+
+   {
+       "page": 2,
+       "limit": 10,
+       "pages": 4,
+       "total": 38,
+       "_links": {
+          "self": {
+             "href": "https://demo.akeneo.com/api/rest/v1/categories?page=2&limit=10"
+          },
+          "first": {
+             "href": "https://demo.akeneo.com/api/rest/v1/categories?page=1&limit=10"
+          },
+          "last": {
+             "href": "https://demo.akeneo.com/api/rest/v1/categories?page=4&limit=10"
+          },
+          "previous": {
+             "href": "https://demo.akeneo.com/api/rest/v1/categories?page=1&limit=10"
+          },
+          "next": {
+             "href": "https://demo.akeneo.com/api/rest/v1/categories?page=3&limit=10"
+          }
+       },
+       "_embedded": {
+          "items": [
+              ...
+          ]
+       }
+   }
+
+.. note::
+    Previous and next keys will not be included if you requested, respectively, the first or the last page.
+
+
+Request format
+--------------
+
+Headers
+~~~~~~~
+
+POST, PUT and PATCH requests must specify a `Content-Type` header set to `application/json`.
+
+Body
+~~~~
+
+.. note::
+  :doc:`/reference/standard_format/index` is used for both sending and receiving data from the API.
```

Comment:
Note exactly, as demonstrated by the response structure from above.
It could be read as "In case of a response, each item in the `items` JSON array follows the standard format.